tsl/test/expected/continuous_aggs_bgw.out

```diff
@@ -486,8 +475,8 @@ view_definition |  SELECT time_bucket(2, test_continuous_agg_table."time") AS ti
 select view_name, completed_threshold, invalidation_threshold, job_status, last_run_duration from timescaledb_information.continuous_aggregate_stats where view_name::text like '%test_continuous_agg_view';
 -[ RECORD 1 ]----------+-------------------------
 view_name              | test_continuous_agg_view
-completed_threshold    | 6
-invalidation_threshold | 6
+completed_threshold    | 
+invalidation_threshold | 12
```

Comment:
Is this 12 because the max bucket is 10 and then you had a -2 end_interval (which means 2 into the future)? Do we want to allow that in that case?

Reply:
resolving as we decided at the meeting that negative values are ok.